eth2/types/src/utils/serde_utils.rs

```diff
@@ -25,6 +25,29 @@ where
     serializer.serialize_str(&hex)
 }
 
+pub fn u32_from_hex_str<'de, D>(deserializer: D) -> Result<u32, D::Error>
+where
+    D: Deserializer<'de>,
+{
+    let s: String = Deserialize::deserialize(deserializer)?;
+
+    u32::from_str_radix(&s.as_str()[2..], 16)
```

Comment:
This looks like it'll raise a panic if `s.len() < 2`.

I can see this happens later in the file as well, from before this PR. Would you mind using the safe `.get()` method here and on those other ones, too please?

Reply:
Duh right, I didn't even look at this code since it was already included in the repo. I'll fix that ASAP :D sharp eye!